boardgamebuddy_api/views.py

```diff
@@ -31,9 +35,13 @@
 @api_view(['GET', 'POST'])
 def user_list(request):
     if request.method == 'GET':
-        users = User.objects.all()
-        serializer = UserSerializer(users, many=True)
-        return JsonResponse(serializer.data, safe=False)
+        if cache.get("all_users") == None:
```

Comment:
Is this optimized for our static user set or dynamic users? I know Prissilla has been working on adding/removing users

Reply:
its not right now.... BUT i think that we could just go with not caching this bc its really not going to make that big of a difference at this scale rn. I think in a future state, we'd have a log in page instead of a screen of all users for scalability

Reply:
@reedhillmar just removed this and pushed the update! 

Have a look and LMK what you think! Can always make some adjustments if we need!